data/JeffProd.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "JeffProd",
+  "type": "personal",
+  "bio": "Coding for fun",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/jeffprod",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "YouTube",
+      "url": "https://www.youtube.com/channel/UCAOizN1F23kLohMWo0f27-Q",
+      "icon": "FaYouTube"
```

Comment:
Please replace it with **" FaYoutube "**  :+1: 
```Suggestion
      "icon": "FaYoutube"
```

Reply:
You need to update your Quick Start picture too : https://user-images.githubusercontent.com/82668196/208292185-743dc114-5699-4b58-bf5c-c5fc74f0248d.png

Reply:
Hey @Tazeg, we are sorry for the confusion arose at your end, but #5484 is already working on these.